en/test_and_ci/integration_testing_mavsdk.md

```diff
@@ -0,0 +1,79 @@
+# Integration Testing using MAVSDK
+
+PX4 can be tested end to end to using integration tests based on [MAVSDK](https://mavsdk.mavlink.io).
+
+The tests are primarily developed against SITL for now and run in continuous integration (CI). However, they are meant to generalize to real tests eventually.
+
+## Install the MAVSDK C++ library
+
+The tests need the MAVSDK C++ library installed system-wide (e.g. in `/usr/lib` or `/usr/local/lib`.
+
+MAVSDK can either be installed as a prebuilt library or alternatively be built from source and installed system-wide.
+
+### Installation of prebuilt library
+
+#### Ubuntu
+
+Download the matching .deb file (whichever is correct for your system) from [MAVSDK releases](https://github.com/mavlink/MAVSDK/releases).
+
+Install it using `dpkg`, e.g.:
+
+```sh
+sudo dpkg -i mavsdk_0.23.0_ubuntu18.04_amd64.deb
+```
+
+#### Fedora
+
+Download the matching .rpm file (whichever is correct for your system) from [MAVSDK releases](https://github.com/mavlink/MAVSDK/releases).
+
+Install it using `rpm`, e.g.:
+
+```sh
+sudo rpm -U mavsdk-0.23.0-1.fc30-x86_64.rpm
+```
+
+#### macOS
+
+Install the library using [brew](https://brew.sh/):
+
+```sh
+brew install mavsdk
```

Comment:
@julianoes Can you install a particular version of the SDK with brew? How?

Reply:
You could get that from the brew docs.